sipreco_purchase/models/purchase_requisition.py

```diff
@@ -87,6 +87,11 @@ def to_inspected(self):
         self.inspected = True
         self.user_inspected_id = self.env.user
 
+    def revert_inspection(self):
+        if self.inspected:
```

Comment:
`for rec in self.filtered('inspected'):
   rec.inspected = False
    rec.user_inspected_id = False`

Reply:
`self.ensure_one()` otra alternativa es usar eso sin el for, mas perecido al `action_draft`